apps/package.json

```diff
@@ -98,7 +98,7 @@
     "eslint-plugin-babel": "^4.1.2",
     "eslint-plugin-mocha": "^4.9.0",
     "eslint-plugin-prettier": "^3.0.1",
-    "eslint-plugin-react": "^5.2.2",
+    "eslint-plugin-react": "7",
```

Comment:
Is it intentional that there is no ^ or ~ here?

Reply:
Yep: `7` [is equivalent to](https://semver.npmjs.com/) `7.x.x`, `^7` or `~7`, and mostly what matters here is the version pinned in yarn.lock.  I guess if we want to get specific here I'd use `>=7.11.0` which is the most recent version with a bugfix to the `react/button-has-type` rule. Opinion on that?

Reply:
Hmm I'm personally a fan of ^7.11.0 or ~7.11.0. It matches our convention, and it's a bit clearer for the next person upgrading this package.

Reply:
`~7.11.0` would only allow patch updates (`7.11.1` is in, `7.12.0` is out).  I'll go with `^7.11.0` which also allows minor version updates.